### PR TITLE
Add TAG's Working Groups section and Green Reviews WG

### PR DESCRIPTION
**Pull request by guidemetothemoon**
This PR adds Green Reviews Working Group to Environmental Sustainability TAG repo and website.

**GitHub issues:** #116 , #64 
**Authors:** @guidemetothemoon, @nikimanoledaki, @leonardpahlke  
**TAG ENV WG Green Reviews Proposal Document:** https://docs.google.com/document/d/18MkLryOSbZpSsgl_ileLbHp_K4xn8TqxMzcHDtNuY_o/edit?usp=sharing

Since Green Reviews Working Group is the first WG to be added, this PR also adds folder structure and information about TAG's Working Groups in the TAG's repo as well as on the TAG's website.
Working Groups section has been added as About subsection on the website:
image

### CODE REVIEW

**Comment by guidemetothemoon** [comment]
Thank you to everyone who has reviewed this proposal until now and came with valuable suggestions and comments. Reviews, comments and suggestions to this PR are very welcome!

File: working-groups/green-reviews/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Green Reviews Working Group
+
+The Green Reviews Working Group operates under CNCF Environmental Sustainability TAG.
```
Comment (nikimanoledaki) [inline working-groups/green-reviews/README.md:3]:
```suggestion
The Green Reviews [Working Group](https://github.com/cncf/toc/tree/main/workinggroups) operates under CNCF Environmental Sustainability TAG.
```

File: README.md [hunk 2]
```diff
@@ -16,6 +16,15 @@ In case you are missing something or is unclear, your contribution is more than
 
 In addition to what you can find in this repository, the TAG Environmental Sustainability maintains the website [sustainability.cncf.io](https://sustainability.cncf.io/) to more easily present information about TAG.
 
+## Working Groups
+
+Following working groups operate under the TAG Environmental Sustainability:
```
Comment (nikimanoledaki) [inline README.md:21]:
```suggestion
Following [working groups](https://github.com/cncf/toc/tree/main/workinggroups) operate under the TAG Environmental Sustainability:
```

**Review by nikimanoledaki — APPROVED** [review]
It's happening! Added a couple of extra refs to the WG definition. Thank you Kristina for bootstrapping the WG structure 😄

File: working-groups/green-reviews/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+## Chairs
+
+- Niki Manoledaki, [@nikimanoledaki](http://github.com/nikimanoledaki)
+- Kristina Devochko, [@guidemetothemoon](http://github.com/guidemetothemoon)
+
+## Meetings and Contact
+
+The Green Reviews Working Group utilizes TAG Environmental Sustainability channels for communication.
+
+- Meeting time and frequency: TBA.
```
Comment (mkorbi) [inline working-groups/green-reviews/README.md:28]: Maybe you can propose here already a time e.g. every other Wednesday or so

Comment (nikimanoledaki) [reply working-groups/green-reviews/README.md:28]: How about the following so that it alternates with the regular meeting? 
```suggestion
- Meeting time and frequency: Every 2nd & 4th Wednesday, 5pm GMT +2:00, 8AM Pacific, 11AM Eastern
```

Comment (guidemetothemoon) [reply working-groups/green-reviews/README.md]: Thank you for the suggestion, Niki and Max, updated with suggested time and frequency.

**Review by roberthstrand — APPROVED** [review]
Looking good 👍

File: working-groups/green-reviews/README.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+## Motivation
+
+CNCF projects are becoming increasingly concerned and aware of the challenges related to environmental sustainability. Their objective is to integrate an environmental sustainability project review in each release cycle. Furthermore, they strive to empower end users to assess the sustainability of the cloud native projects within their systems. 
+
+The purpose of the Green Reviews Working Group is to help CNCF projects to review their cloud native sustainability footprint.
+
+To support CNCF projects, project maintainers and contributors, there is a need for a systematic approach to assess every project's sustainability footprint. Implementing a structured review process for this purpose would greatly benefit the CNCF ecosystem, establishing it as a best practice. Such a process would enable stakeholders to comprehensively evaluate and enhance the sustainability of CNCF projects, promoting their long-term viability and environmental responsibility. 
+
+Green Reviews Working Group empowers the CNCF Environmental Sustainability TAG to pursue its goals of promoting environmentally sustainable practices in the CNCF ecosystem.
```
Comment (leonardpahlke) [inline working-groups/green-reviews/README.md:17]: Should we move this to the charter doc? Why did we split it up?

File: working-groups/green-reviews/README.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+## Chairs
```
Comment (leonardpahlke) [inline working-groups/green-reviews/README.md:19]: Since this WG will persist, @catblade @mkorbi we need to how long leads serve in this role and how followup elections look like.

File: working-groups/green-reviews/README.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+- Niki Manoledaki, [@nikimanoledaki](http://github.com/nikimanoledaki)
+- Kristina Devochko, [@guidemetothemoon](http://github.com/guidemetothemoon)
```
Comment (leonardpahlke) [inline working-groups/green-reviews/README.md:22]: we need to add a date after the name (start of the term)

File: website/content/about/working-groups.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
+---
+title: Working Groups
+description: This section provides information about working groups that operate under the TAG Environmental Sustainability.
+slug: working-groups
+---
+
+The purpose of working groups are to study and report on a particular question and make recommendations based on its findings. You can read more about the intention of working groups and the overall process behind submitting a working group proposal on the Technical Oversight Committee (TOC) GitHub: [CNCF Working Groups](https://github.com/cncf/toc/blob/main/workinggroups/README.md).
```
Comment (leonardpahlke) [inline website/content/about/working-groups.md:7]:
```suggestion
TAG Environmental Sustainability has working groups and teams to effectively collaborate and coordinate efforts towards achieving common goals and objectives, as well as to distribute tasks and responsibilities among its members for efficient execution. This approach allows for a more streamlined and organized approach to managing complex projects within the CNCF ecosystem. You can read more about the intention of working groups and the overall process behind submitting a working group proposal on the Technical Oversight Committee (TOC) GitHub: [CNCF Working Groups](https://github.com/cncf/toc/blob/main/workinggroups/README.md).
```
reused the TAG Contrib Strat phrasing here [link](https://github.com/cncf/tag-contributor-strategy#working-groups-and-teams)

File: working-groups/green-reviews/README.md (hunk 5 above)
Comment (catblade) [reply working-groups/green-reviews/README.md:19]: It makes sense for there to be a commitment as a lead.  I'd recommend we do at least 6 months.  @TheFoxAtWork  what is the normal cadence of assignments for leads of WGs?

Comment (guidemetothemoon) [reply working-groups/green-reviews/README.md:9]: Yes, thanks for bringing this up. Depending on what the process is we can either add an abstract describing this or refer to the same process that we have in the TAG if it can be re-used here as well. I can update the content once we get some additional information here.

File: working-groups/green-reviews/README.md (hunk 4 above)
Comment (guidemetothemoon) [reply working-groups/green-reviews/README.md]: @leonardpahlke my initial thought was to keep some information about the background for the working group in the README, because that's normally the first place people will see first. But you have a valid point, now that I take a second look at it I see that the introduction abstract provides enough information to begin with. Motivation can be moved to the charter doc where more details can be found if you want to learn more. Updated PR for the same.

File: working-groups/green-reviews/charter.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+- Collaborate with CNCF project maintainers to produce meaningful sustainability footprint reviews.
+- Communicate the results of the projects’ cloud native sustainability reviews.
+
+
+## Non-Goals
+
+- Build tooling for measuring or monitoring sustainability-related metrics from scratch.
+- Contribute code to existing tooling and cloud native sustainability projects such as bug fixes or features.
+- Providing any consultancy or guidance that goes beyond facilitating the projects’ sustainability footprint reviews.
+
```
Comment (leonardpahlke) [inline working-groups/green-reviews/charter.md:38]:
```suggestion
- Work on the annual CNCF sandbox project reviews as they are managed separately from this working group.
```

File: working-groups/green-reviews/README.md (hunk 5 above)
Comment (guidemetothemoon) [reply working-groups/green-reviews/README.md]: @leonardpahlke @catblade @nikimanoledaki @mkorbi I have added a document for the Working Group Chair nomination process as part of this PR where I used tech lead proposal doc as a foundation with further adjustments and simplifications for this specific use case - can you please take a look when you have time and let me know if it requires any further adjustments? Thanks! 😊

Comment (leonardpahlke) [reply working-groups/green-reviews/README.md]: LGTM

File: governance/working-group-chair-proposal-process.md [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+
+TAG Environmental Sustainability Working Group Chairs operate as a team alongside the TAG's co-chairs and technical leads. 
+They provide guidance and directions to the working group members and ensure that the working group's efforts are in line with it's mission and values, as well as the overall goals of the TAG. They facilitate the working group's activities and coordinate its members' efforts. They schedule working group meetings, set meeting agenda, and maintain effective communication to ensure smooth collaboration among the working group's participants.
+Finally, they provide regular updates to the TAG, keeping the TAG leadership team and members informed about the group's progress and deliverables.
+
+## Caveats of Working Group Chairs
+
+* The position of a working group chair is not a paid position (TAG co-chairs and tech leads are also not paid positions)
+* Working group Chairs [have responsibilities](roles.md) to the Environmental Sustainability TAG, the leadership team, and the TOC
+* Nominations by the community, with endorsements by 1 or more working group/TAG co-chairs
```
Comment (TheFoxAtWork) [inline governance/working-group-chair-proposal-process.md:11]:
```suggestion
* Nominations made by the community, need endorsements by 1 or more working group/TAG co-chairs, or tech leads
```
Are tech leads allowed to provide endorsement?

File: governance/working-group-chair-proposal-process.md [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+Any individual who is a member of the Environmental Sustainability TAG list or the tag-environmental-sustainability slack channel may nominate another individual.
+
+The working group co-chairs reserve the right to endorse a nominee by the community based on:
+
+* Gender diversity
+* Company diversity
+* Geo diversity
+* Amount of nominations from different companies, people, regions for a single individual
+* Existing body of work
+* Justification provided during nomination
```
Comment (TheFoxAtWork) [inline governance/working-group-chair-proposal-process.md:31]: Recommend adjusting this slightly. The current text states that endorsements may come from the WG chairs or TAG chairs (recommend also including tech leads). I would reword this to the effect that endorsement and selection of individuals will consider the following to ensure appropriate community representation and technical acumen.

File: governance/working-group-chair-proposal-process.md [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+
+* Subject "Working Group $NAME-OF-WORKING-GROUP chair nomination: $NAME-OF-NOMINEE"
+* CC the nominee
+* Justification
+
+#### Endorsement of community nominees
+
+Working group co-chairs are responsible for providing endorsement of a community nominee to show concurrence and support.
+
+If no nominees are submitted by the community or from working group/TAG co-chairs, either as a result of lack of participation or lack of requirements, the working group co-chairs must endeavor to nominate an individual(s) that meet as many of the requirements as exist.
```
Comment (TheFoxAtWork) [inline governance/working-group-chair-proposal-process.md:50]: or perhaps consider the activity of the group and whether it needs to continue.  If there are no individuals interested in leading, not much work can be achieved.

**Review by TheFoxAtWork — APPROVED** [review]
Looks good overall, some questions on the nomination/ endorsement process

File: working-groups/green-reviews/charter.md [hunk 12]
```diff
@@ -0,0 +1,53 @@
...
+- Provide resources through a review process and methodology to help CNCF projects to evaluate their sustainability footprint[^1] and integrate environmental sustainability practices. 
+- Collaborate with CNCF project maintainers to produce meaningful sustainability footprint reviews.
+- Communicate the results of the projects’ cloud native sustainability reviews.
+
+
+## Non-Goals
+
+- Build tooling for measuring or monitoring sustainability-related metrics from scratch.
+- Contribute code to existing tooling and cloud native sustainability projects such as bug fixes or features.
```
Comment (nikhita) [inline working-groups/green-reviews/charter.md:36]: For my understanding -- can you clarify the amount of tooling (maybe through some examples?) that will be in scope for the WG? I'm a little confused because:

The `Scope` section calls out:

> Create minimal tooling to conduct CNCF project sustainability footprint reviews, for example, to automate certain processes, if existing tooling does not meet requirements, and if the CNCF Environmental Sustainability TAG formally approved this.

and `Deliverables` mention:

> Develop testing infrastructure to utilise existing tooling to measure the sustainability footprint of CNCF projects.

Does this involve writing tooling to build the test infra?

> Provide visualisation to communicate the outcomes of the sustainability footprint CNCF project reviews, for example, through dashboards for carbon and energy monitoring.

Will existing tooling be used to build dashboards or does this involve building additional tooling?

**Review by nikhita — COMMENTED** [review]
Left a question about scope but looks great otherwise!

Thanks for including an explicit process around nomination too 👍

File: governance/working-group-chair-proposal-process.md (hunk 9 above)
Comment (leonardpahlke) [reply governance/working-group-chair-proposal-process.md:11]: I dont see why not. +1 adding it

Comment (guidemetothemoon) [reply governance/working-group-chair-proposal-process.md]: +1 from me as well, approved suggested change for this.

File: governance/working-group-chair-proposal-process.md (hunk 11 above)
Comment (guidemetothemoon) [reply governance/working-group-chair-proposal-process.md]: @TheFoxAtWork @raravena80 thanks for the feedback! I wasn't able to find any official process for working group or TAG archival so I created a separate file in `governance` folder that outlines a baseline description of working group archival process.
I have linked it in the working group `charter.md`, as well as in the **Process of nominations** section of `working-group-chair-proposal-process.md`, instead of having an endorsement section. Please let me know if you disagree or if anything should be updated.
@leonardpahlke @nikimanoledaki FYI.

File: governance/working-group-chair-proposal-process.md (hunk 10 above)
Comment (guidemetothemoon) [reply governance/working-group-chair-proposal-process.md]: Thanks, updated this section!

File: working-groups/green-reviews/charter.md (hunk 12 above)
Comment (leonardpahlke) [reply working-groups/green-reviews/charter.md:35]: Right, good call out, we talked about tooling for a bit during the two [WG discussion meetings](https://www.youtube.com/watch?v=bAZEWr85Ggg&list=PL_3_oSSuNpD4DDVxVF4zx8HklbOUP9HFP). For now, the focus lies on the discovery part, which is essentially about understanding how we want to review the sustainability footprint of CNCF projects. As we don't know the step-by-step process yet, we are unsure about the automation options we have down the road. It's highly likely that we will benefit from automation, that's why we mention it in the first place in the charter. We expressed worries during our calls that this may lead to working on an entire test suite or something similar, which would lead off from the core course of this WG. So we open up the option of developing automation tooling, but we limit it by phrasing it "minimal automation tooling". With minimal automation tooling, we follow the idea of adopting existing libraries and projects if possible and only, for example, add a script or small-sized programs on top of. Maybe we can clarify this, what do you think?

Comment (nikhita) [reply working-groups/green-reviews/charter.md:35]: > Maybe we can clarify this, what do you think?

Sounds good!

Comment (leonardpahlke) [reply working-groups/green-reviews/charter.md:35]: cc @nikimanoledaki @guidemetothemoon

Comment (guidemetothemoon) [reply working-groups/green-reviews/charter.md:35]: @nikhita @leonardpahlke @nikimanoledaki thanks for providing detailed reflections, I totally agree that this point needed additional clarification. Based on this discussion I've attempted to improve clarification of the respective points both in the Scope and Deliverables section. Please take a look when you have time and let me know if anything else should be changed or improved😊

Comment (nikhita) [reply working-groups/green-reviews/charter.md:34]: @guidemetothemoon thanks! LGTM :shipit:

**Comment by mkorbi** [comment]
LGTM 
Let's go!

**Review by TheFoxAtWork — APPROVED** [review]
This looks absolutely wonderful and well defined. Thank you!
